org.gnome.Cheese.yml

```diff
@@ -0,0 +1,75 @@
+id: org.gnome.Cheese
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+runtime-version: 3.34
+command: cheese
+tags:
+  - 3.34.0
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --device=all
+  - --metadata=X-DConf=migrate-path=/org/gnome/Cheese/
+  - --talk-name=ca.desrt.dconf
```

Comment:
```suggestion
```

Reply:
@TingPing had provided the background for removing these earlier in IRC but I missed this and on other when actually making the changes in the file. Fixed in 23bd80cf465354b0ba859af990ea44eaf36511f2. Thanks.

Reference: https://github.com/flathub/org.gnome.Totem/pull/36/files